### ISSUE TITLE
SQL error: Unknown column 'title' in 'order clause'

### ISSUE THREAD

**Issue by arifje** [putyourlightson/craft-campaign#469]
### Bug Report

I am getting an SQL error when displaying some mailing lists in the users profile page, caused by **getMailingLists()** function;

Snippet of my code;

```
{% set userContact = craft.campaign.contacts.userId(currentUser.id).one() %}
{% set userMailingLists = userContact ? userContact.getMailingLists() : [] %}
```

Error:

```
Database Exception – [yii\db\Exception](https://www.yiiframework.com/doc-2.0/yii-db-exception.html)
SQLSTATE[42S22]: Column not found: 1054 Unknown column 'title' in 'order clause'
The SQL being executed was: SELECT `mailingListId`
FROM (SELECT `campaign_contacts_mailinglists`.*
FROM `campaign_contacts_mailinglists`
INNER JOIN `elements` `contactElement` ON `contactElement`.`id` = `contactId`
INNER JOIN `elements` `mailingListElement` ON `mailingListElement`.`id` = `mailingListId`
WHERE (`contactElement`.`draftId` IS NULL) AND (`contactElement`.`dateDeleted` IS NULL) AND (`mailingListElement`.`dateDeleted` IS NULL)) `subquery`
WHERE `contactId`=40332
ORDER BY `title`
```

<img width="1016" alt="Screenshot 2024-04-20 at 20 54 14" src="https://github.com/putyourlightson/craft-campaign/assets/1286410/28111e48-6457-4dad-847e-bafdab0d812a">

### Plugin Version

2.15.1

### Craft CMS Version

4.8.9

**Comment by bencroker** [comment]
Thanks for reporting this. Fixed in https://github.com/putyourlightson/craft-campaign/commit/3ea883d75e0992c899d5f05859ecc7dde977c990 for the next release.

**Comment by bencroker** [comment]
Released in [2.15.2](https://github.com/putyourlightson/craft-campaign/releases/tag/2.15.2).
